test/memory_coloring_test.cpp

```diff
@@ -608,4 +640,47 @@ TEST_CASE(literal_test)
     CHECK(lit == result);
 }
 
+TEST_CASE(concurrent_test)
+{
+    migraphx::program p;
+    auto in = p.add_parameter("0", migraphx::shape{migraphx::shape::float_type, {40}});
+    auto a1 = add_alloc(p, {migraphx::shape::float_type, {40}});
+    auto p1 = p.add_instruction(pass_op{}, a1, in);
+    p.insert_instruction(p1, set_stream{0});
+    p1->set_stream(0);
```

Comment:
Shouldn't you be inserting a `set_stream` instruction?

Reply:
As mentioned in earlier comments, "dom_info" get stream from instruction.